### ISSUE TITLE
Provide something like ActiveRecord::Base.connection.quote

### ISSUE THREAD

**Issue by robacarp** [amberframework/granite-orm#122]
Hopefully this is buried and salty to use, but I think the ORM should provide it. Especially since this ORM is still developing (as in #121), it would be nice to be able to manually escape a string where no other option exists.

**Comment by drujensen** [comment]
@robacarp will this work for you or do you think we still need the `quote` capability?

**Comment by robacarp** [comment]
I think Granite should still have quote capability. That kind of depth is the right amount of depth though because it should be raising flags of warning when used. `Model.adapter.database.quote "string"` is at least [salty enough](http://catb.org/jargon/html/S/syntactic-salt.html) to be noticed by code reviewers and to cause a bit of rethinking by developers.

It's hopefully not _common_ but I have used ActiveRecord's adapter quote mechanism a time or two. I can't remember why I couldn't use a parameterized expression, but I'm sure I would have tried that road first.

Maybe it's worth waiting on the need to appear though?

**Comment by faustinoaq** [comment]
Is this already being fixed by #150 ? or duplicate of #149 ?

**Comment by drujensen** [comment]
Closing this issue.
